data_sender.py

```diff
@@ -1,11 +1,34 @@
-from multiprocessing import Process
+import time
+import serial
+import struct
 from video_live_generator import VideoLiveGenerator
+import color_detector
+from ball_detection import BallDetection
 
-driver_camera = VideoLiveGenerator(1)
-auto_camera = VideoLiveGenerator(2)
 
-driver_station_feed = Process(target=driver_camera.get_next_frame(), name="driver station camera feed")
-autonomous_feed = Process(target=auto_camera.get_next_frame(), name="autonomous camera feed")
+auto_camera = VideoLiveGenerator(1)
+# track the time to close the camera once the match is over
+start_time = time.time()
 
-driver_station_feed.start()
-autonomous_feed.start()
+# open the serial port
+ser = serial.Serial('/dev/')
+print(ser.name)
+
+
+# run the cv code for 150 seconds (15 for autonomous + 125 for TeleOp)
```

Comment:
We should run the CV code indefinitely. The robot gets powered on when we put it on the field, and the Jetson will start running CV then. We don't know how long it will be until the match actually starts.

Reply:
Thank you for the feedback. I have updated it.